pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "hatchling.build"
 
 [project]
 name = "metricflow"
-version = "0.202.0.dev0"
+version = "0.202.0.dev1"
```

Comment:
Let's revert this for now and update separately, we've got a bunch of these version update changes flying around and it messes with the hatch build -> deploy process.